enrichment-mongo/src/main/java/eu/europeana/enrichment/mongo/service/PersistentTopicEntityServiceImpl.java

```diff
@@ -0,0 +1,48 @@
+package eu.europeana.enrichment.mongo.service;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import eu.europeana.enrichment.common.commons.AppConfigConstants;
+import eu.europeana.enrichment.model.TopicEntity;
+import eu.europeana.enrichment.model.TopicModel;
+import eu.europeana.enrichment.model.impl.TopicEntityImpl;
+import eu.europeana.enrichment.mongo.dao.TopicEntityDao;
+
+@Service(AppConfigConstants.BEAN_ENRICHMENT_PERSISTENT_TOPIC_ENTITY_SERVICE)
+public class PersistentTopicEntityServiceImpl implements PersistentTopicEntityService{
+	
+	
+	@Autowired
+	private TopicEntityDao topicEntityDao;
+
+	@Override
+	public void saveTopicEntity(TopicEntity topicEntity) {
```

Comment:
please use only save, update, find, delete in the method names to reduce redundancy